api-docs/swagger.yml

```diff
@@ -253,6 +253,87 @@ paths:
           description: Not Found
         '500':
           description: Internal Server Error
+  '/system/secrets':
+    get:
+      summary: 'Get a list of secret names and metadata from the provider'
+      consumes:
+      - application/json
+      produces:
+      - application/json
+      responses:
+        '200':
+          description: List of submitted secrets.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/ListSecretsResponse'
```

Comment:
All refs should point to `'#/definitions/Secret'`

Reply:
Agreed. PTAL in my second commit